community/thrift/APKBUILD

```diff
@@ -0,0 +1,222 @@
+    mv "$tpkgdir"/usr/include/${pkgname}/c_glib "$subpkgdir"/usr/include/${pkgname}
+    install -d "$subpkgdir"/usr/lib/pkgconfig
+    mv "$tpkgdir"/usr/lib/pkgconfig/${pkgname}_c_glib.pc "$subpkgdir"/usr/lib/pkgconfig
+}
+
+cpp() {
+    local tpkgdir="${pkgdir}-tmp"
+    pkgdesc="${pkgname} c++ bindings"
+    depends="boost libevent openssl zlib"
+    install -d "$subpkgdir"/usr/lib
+    mv \
+        "$tpkgdir"/usr/lib/lib${pkgname}-${pkgver}.so \
+        "$tpkgdir"/usr/lib/lib${pkgname}z-${pkgver}.so \
+        "$tpkgdir"/usr/lib/lib${pkgname}nb-${pkgver}.so \
+        "$subpkgdir"/usr/lib    
+}
+
+cppdev() {
+    local tpkgdir="${pkgdir}-tmp"
+    pkgdesc="${pkgname} c++ bindings (development files)"
+    depends="openssl zlib libc-dev"
+    install -d "$subpkgdir"/usr/lib
+    mv \
+        "$tpkgdir"/usr/lib/lib${pkgname}.a \
+        "$tpkgdir"/usr/lib/lib${pkgname}z.a \
+        "$tpkgdir"/usr/lib/lib${pkgname}nb.a \
+        "$subpkgdir"/usr/lib
+    install -d "$subpkgdir"/usr/include/${pkgname}
+    mv \
+        "$tpkgdir"/usr/include/${pkgname} \
+        "$subpkgdir"/usr/include
+    install -d "$subpkgdir"/usr/lib/pkgconfig
+    mv \
+        "$tpkgdir"/usr/lib/pkgconfig/${pkgname}-z.pc \
+        "$tpkgdir"/usr/lib/pkgconfig/${pkgname}.pc \
+        "$tpkgdir"/usr/lib/pkgconfig/${pkgname}-nb.pc \
+        "$subpkgdir"/usr/lib/pkgconfig
+}
+
+#go() {
+#    cd "$builddir"
+#    pkgdesc="${pkgname} go bindings"
+#    depends="go"
+#}
+
+#java() {
+#    cd "$builddir"
+#    pkgdesc="${pkgname} java bindings"
+#    depends="java"
+#}
+
+lua() {
+    local tpkgdir="${pkgdir}-tmp"
+    pkgdesc="${pkgname} lua bindings"
+    depends="lua${_luaver} lib${pkgname}"
+    install -d "$subpkgdir"/usr/lib
+    mv "$tpkgdir"/usr/lib/liblua*.so* "$subpkgdir"/usr/lib
+    install -d "$subpkgdir"/usr/share/lua/${_luaver}
+    cp "$builddir"/lib/lua/*.lua "$subpkgdir"/usr/share/lua/${_luaver} 
+}
+
+# XXX: @CF: 20180715: this works fine, but users should probably just stick with "npm install thrift"
+#nodejs() {
+#    cd "$builddir"
+#    pkgdesc="${pkgname} nodejs bindings"
+#    install -d "$subpkgdir"/usr/lib/node
+#    mv lib/nodejs/lib/${pkgname} node_modules/node-int64 node_modules/ws node_modules/q "$subpkgdir"/usr/lib/node
+#}
+
+perl() {
+    local tpkgdir="${pkgdir}-tmp"
+    arch="noarch"
+    pkgdesc="${pkgname} perl bindings"
+    depends="perl-bit-vector perl-class-accessor openssl"
+    install -d "$subpkgdir"/usr/lib
+    mv "$tpkgdir"/usr/lib/perl5 "$subpkgdir"/usr/lib
+}
+
+py() {
+    local tpkgdir="${pkgdir}-tmp"
+    pkgdesc="${pkgname} python bindings"
+    depends="python2 py-openssl py2-backports.ssl_match_hostname py2-six"
+    install -d "$subpkgdir"/usr/lib
+    mv "$tpkgdir"/usr/lib/python2.7 "$subpkgdir"/usr/lib
+}
+
+#ruby() {
+#    cd "$builddir"
+#    pkgdesc="${pkgname} ruby bindings"
+#    depends="ruby"
+#}
+
+#rust() {
+#    cd "$builddir"
+#    pkgdesc="${pkgname} rust bindings"
+#    depends="rust"
+#}
```

Comment:
You don't need to comment out these split functions.

Reply:
The'yre currently unimplimented anyway.